docs/src/main/asciidoc/transaction.adoc

```diff
@@ -383,19 +391,19 @@ The default value is `false`.
 * `table-prefix` (string): Specify the prefix for a related table name.
 The default value is `quarkus_`.
 
-// This paragraph will differ from the RHBQ docs in the future since it is not supported in the product yet.
-Additional information:
+For more configuration information, see the *Narayana JTA - Transaction manager* section of the Quarkus xref:all-config.adoc[All configuration options] reference.
+
+.Additional information:
 
 * You can manually create the transaction log table during the initial setup by setting the `create-table` property to `true`.
 
 * JDBC datasources and ActiveMQ Artemis allow the enlistment and automatically register the `XAResourceRecovery`.
 
 ** JDBC datasources is part of `quarkus-agroal`, and it needs to use `quarkus.datasource.jdbc.transactions=XA`.
 ** ActiveMQ Artemis is part of `quarkus-pooled-jms`, and it needs to use `quarkus.pooled-jms.transaction=XA`.
-+
-For more information, see link:https://issues.redhat.com/browse/CEQ-4878[CEQ-4878].
 
-* If your application employs eXtended Architecture (XA) transactions, enable the transaction crash recovery feature by using `quarkus.transaction-manager.enable-recovery=true`, to safeguard your data in the event of application crashes or failures. The default value for XA Recovery is `false`.
+* If your application employs JTA transactions style, enable the transaction crash recovery feature by using `quarkus.transaction-manager.enable-recovery=true`, to safeguard your data in the event of application crashes or failures.
+The default value for JTA recovery is `false`.
```

Comment:
Although the rendering of the document appears to add the sentence `The default value for JTA recovery is false.` on the same line...I'd like to suggest to put `The default value for JTA recovery is false.` on the same line.
```suggestion
* If your application employs JTA transactions style, enable the transaction crash recovery feature by using `quarkus.transaction-manager.enable-recovery=true`, to safeguard your data in the event of application crashes or failures. The default value for JTA recovery is `false`.
```

Reply:
It is already clear that the guide is for JTA transactions, after all it is its raison d'etre. So just use the term `recovery` since JTA is implied - also see my earlier comment about the terminology around transactions.